promql/engine.go

```diff
@@ -2760,8 +2771,14 @@ func (ev *evaluator) aggregation(e *parser.AggregateExpr, grouping []string, par
 				if group.histogramMean != nil {
 					left := s.H.Copy().Div(float64(group.groupCount))
 					right := group.histogramMean.Copy().Div(float64(group.groupCount))
-					toAdd := left.Sub(right)
-					group.histogramMean.Add(toAdd)
+					toAdd, err := left.Sub(right)
+					if err != nil {
+						panic(err)
+					}
+					_, err = group.histogramMean.Add(toAdd)
+					if err != nil {
+						panic(err)
```

Comment:
We should not panic in the engine, instead ignore the value in the addition. Possibly throw a warning.